Update AWS RDS procedure (#2727)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-rds.md

| Rds-Cpu-Credit   | Cloud-Aws-Rds-Cpu-Credit-Api-custom   | Contrôle permettant de vérifier l'utilisation des crédits CPU |
| Rds-Cpu-Usage    | Cloud-Aws-Rds-Cpu-Usage-Api-custom    | Contrôle permettant de vérifier l'utilisation CPU             |
| Rds-Network      | Cloud-Aws-Rds-Network-Api-custom      | Contrôle l'utilisation du réseau                              |
| Rds-Queries      | Cloud-Aws-Rds-Queries-Api-custom      | Contrôle le nombre de requêtes effectuées sur l'instance      |
| Rds-Transactions | Cloud-Aws-Rds-Transactions-Api-custom | Contrôle le nombre de transaction effectuées sur l'instance   |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Rds-Instance-Aurora-custom** est utilisé.

</TabItem>
<TabItem value="Cloud-Aws-Rds-Instance-MSSQL-custom" label="Cloud-Aws-Rds-Instance-MSSQL-custom">

| Alias           | Modèle de service                    | Description                                                   |
|:----------------|:-------------------------------------|:--------------------------------------------------------------|

> | Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexion à l'instance                  |

Reviewer: ```suggestion
| Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexions à l'instance                  |
```

| Rds-Cpu-Credit  | Cloud-Aws-Rds-Cpu-Credit-Api-custom  | Contrôle permettant de vérifier l'utilisation des crédits CPU |
| Rds-Cpu-Usage   | Cloud-Aws-Rds-Cpu-Usage-Api-custom   | Contrôle permettant de vérifier l'utilisation CPU             |
| Rds-Network     | Cloud-Aws-Rds-Network-Api-custom     | Contrôle l'utilisation du réseau                              |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Rds-Instance-MSSQL-custom** est utilisé.

</TabItem>
<TabItem value="Cloud-Aws-Rds-Instance-MariaDB-custom" label="Cloud-Aws-Rds-Instance-MariaDB-custom">

| Alias           | Modèle de service                    | Description                                                   |
|:----------------|:-------------------------------------|:--------------------------------------------------------------|

> | Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexion à l'instance                  |

Reviewer: ```suggestion
| Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexions à l'instance                  |
```

| Rds-Cpu-Credit  | Cloud-Aws-Rds-Cpu-Credit-Api-custom  | Contrôle permettant de vérifier l'utilisation des crédits CPU |
| Rds-Cpu-Usage   | Cloud-Aws-Rds-Cpu-Usage-Api-custom   | Contrôle permettant de vérifier l'utilisation CPU             |
| Rds-Diskio      | Cloud-Aws-Rds-Diskio-Api-custom      | Contrôle l'utilisation des écritures                          |
| Rds-Network     | Cloud-Aws-Rds-Network-Api-custom     | Contrôle l'utilisation du réseau                              |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Rds-Instance-MariaDB-custom** est utilisé.

</TabItem>
<TabItem value="Cloud-Aws-Rds-Instance-MySQL-custom" label="Cloud-Aws-Rds-Instance-MySQL-custom">

| Alias           | Modèle de service                    | Description                                                   |
|:----------------|:-------------------------------------|:--------------------------------------------------------------|

> | Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexion à l'instance                  |

Reviewer: ```suggestion
| Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexions à l'instance                  |
```

| Rds-Cpu-Credit  | Cloud-Aws-Rds-Cpu-Credit-Api-custom  | Contrôle permettant de vérifier l'utilisation des crédits CPU |
| Rds-Cpu-Usage   | Cloud-Aws-Rds-Cpu-Usage-Api-custom   | Contrôle permettant de vérifier l'utilisation CPU             |
| Rds-Diskio      | Cloud-Aws-Rds-Diskio-Api-custom      | Contrôle l'utilisation des écritures                          |
| Rds-Network     | Cloud-Aws-Rds-Network-Api-custom     | Contrôle l'utilisation du réseau                              |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Rds-Instance-MySQL-custom** est utilisé.

</TabItem>
<TabItem value="Cloud-Aws-Rds-Instance-Oracle-custom" label="Cloud-Aws-Rds-Instance-Oracle-custom">

| Alias           | Modèle de service                    | Description                                                   |
|:----------------|:-------------------------------------|:--------------------------------------------------------------|

> | Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexion à l'instance                  |

Reviewer: ```suggestion
| Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexions à l'instance                  |
```

| Rds-Cpu-Credit  | Cloud-Aws-Rds-Cpu-Credit-Api-custom  | Contrôle permettant de vérifier l'utilisation des crédits CPU |
| Rds-Cpu-Usage   | Cloud-Aws-Rds-Cpu-Usage-Api-custom   | Contrôle permettant de vérifier l'utilisation CPU             |
| Rds-Network     | Cloud-Aws-Rds-Network-Api-custom     | Contrôle l'utilisation du réseau                              |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Aws-Rds-Instance-Oracle-custom** est utilisé.

</TabItem>
<TabItem value="Cloud-Aws-Rds-Instance-PostgreSQL-custom" label="Cloud-Aws-Rds-Instance-PostgreSQL-custom">

| Alias           | Modèle de service                    | Description                                                   |
|:----------------|:-------------------------------------|:--------------------------------------------------------------|

> | Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexion à l'instance                  |

Reviewer: ```suggestion
| Rds-Connections | Cloud-Aws-Rds-Connections-Api-custom | Contrôle le nombre de connexions à l'instance                  |
```

</TabItem>
</Tabs>

### Règles de découverte

#### Découverte d'hôtes

| Nom de la règle | Description                       |
|:----------------|:----------------------------------|
| Amazon AWS RDS  | Discover Amazon AWS RDS instances |

Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.

### Métriques & statuts collectés

Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.

<Tabs groupId="sync">
<TabItem value="Rds-Connections" label="Rds-Connections">

| Métrique            | Description                             | Unité  |
| :------------------ | :-------------------------------------- |:------ |
| DatabaseConnections | Number of connections to the database.  | Count  |

</TabItem>
<TabItem value="Rds-Cpu-Credit" label="Rds-Cpu-Credit">

| Métrique         | Description                                                | Unité                   |                   
| :--------------- | :--------------------------------------------------------- |:----------------------- |

> | CPUCreditBalance | Balance of allocated CPU credit to this type of instance.  | Credits (vCPU-minutes)  |

Reviewer: ```suggestion
| CPUCreditBalance | Balance of allocated CPU credit for this type of instance.  | Credits (vCPU-minutes)  |
```

> | CPUCreditUsage   | Number of CPU credit consumed.                             | Credits (vCPU-minutes)  |

Reviewer: ```suggestion
| CPUCreditUsage   | Number of CPU credits consumed.                             | Credits (vCPU-minutes)  |
```

> | DiskQueueDepth  | The number of outstanding IOs (read/write requests) waiting to access the disk. | Count        |

Reviewer: ```suggestion
| DiskQueueDepth  | The number of outstanding I/Os (read/write requests) waiting to access the disk. | Count        |
```

</TabItem>
<TabItem value="Rds-Instance-Status" label="Rds-Instance-Status">

| Métrique               | Unité |
|:-----------------------|:------|
| total-available        | N/A   |
| total-failed           | N/A   |
| total-backing-up       | N/A   |
| total-maintenance      | N/A   |
| total-stopped          | N/A   |
| total-storage-full     | N/A   |
| *aws_instances*#status | N/A   |

</TabItem>
<TabItem value="Rds-Network" label="Rds-Network">

| Métrique                  | Description                                                                                                                          | Unité         |
| :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------- |:------------- |

> | NetworkReceiveThroughput  | The incoming traffic on the DB instance, including both customer db traffic and AWS/RDS traffic used for monitoring and replication. | Bytes/Second  |

Reviewer: ```suggestion
| NetworkReceiveThroughput  | The incoming traffic on the DB instance, including both customer DB traffic and AWS/RDS traffic used for monitoring and replication. | Bytes/Second  |
```

> | NetworkTransmitThroughput | The outgoing traffic on the DB instance, including both customer db traffic and AWS/RDS traffic used for monitoring and replication. | Bytes/Second  |

Reviewer: ```suggestion
| NetworkTransmitThroughput | The outgoing traffic on the DB instance, including both customer DB traffic and AWS/RDS traffic used for monitoring and replication. | Bytes/Second  |
```

</TabItem>
<TabItem value="Rds-Queries" label="Rds-Queries">

| Métrique         | Description                                                                                             |
| :--------------- | :------------------------------------------------------------------------------------------------------ |

> | Queries          | The average number of queries executed per second **(Only available on Aurora MySQL)**                  |

Reviewer: ```suggestion
| Queries          | The average number of queries executed per second. **(Only available on Aurora MySQL)**                  |
```

> | InsertThroughput | The average number of insert queries per second **(Only available on Aurora MySQL)**                    |

Reviewer: ```suggestion
| InsertThroughput | The average number of insert queries per second. **(Only available on Aurora MySQL)**                    |
```

> | DMLThroughput    | The average number of DataModificationLanguage requests per second **(Only available on Aurora MySQL)** |

Reviewer: ```suggestion
| DMLThroughput    | The average number of DataModificationLanguage requests per second. **(Only available on Aurora MySQL)** |
```

</TabItem>
<TabItem value="Rds-Storage" label="Rds-Storage">

| Métrique                       | Unité |
|:-------------------------------|:------|
| storage.space.free.bytes       | B     |
| storage.space.usage.percentage | %     |
| memory.free.bytes              | B     |

</TabItem>
<TabItem value="Rds-Transactions" label="Rds-Transactions">

| Métrique            | Description                                                                                                                                                                                                                  |
| :------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | ActiveTransactions  | The average number of current transactions executing on an Aurora database instance per second. **(Only available on Aurora MySQL) Set innodb\_monitor\_enable='all' in the DB parameter group for a specific DB instance.** |

Reviewer: ```suggestion
| ActiveTransactions  | The average number of current transactions executing on an Aurora database instance per second. **(Only available on Aurora MySQL.) Set innodb\_monitor\_enable='all' in the DB parameter group for a specific DB instance.** |
```

> | BlockedTransactions | The average number of transactions in the database that are blocked per second **(Only available on Aurora MySQL)**                                                                                                          |

Reviewer: ```suggestion
| BlockedTransactions | The average number of transactions in the database that are blocked per second. **(Only available on Aurora MySQL)**                                                                                                          |
```

> | CommitLatency       | The amount of latency for commit operations, in milliseconds **(Only available on Aurora MySQL and Postgres)**                                                                                                               |

Reviewer: ```suggestion
| CommitLatency       | The amount of latency for commit operations, in milliseconds. **(Only available on Aurora MySQL and Postgres)**                                                                                                               |
```

> | CommitThroughput    | The average number of commit operations per second **(Only available on Aurora MySQL and Postgres)**                                                                                                                         |

Reviewer: ```suggestion
| CommitThroughput    | The average number of commit operations per second. **(Only available on Aurora MySQL and Postgres)**                                                                                                                         |
```

| Métrique        | Description                                                                                                                                                    |
| :-------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| VolumeReadIOPs  | The number of billed read I/O operations from a cluster volume, reported at 5-minute intervals. **(Only available on Aurora MySQL and Postgres)**              |
| VolumeWriteIOPs | The number of write disk I/O operations to the cluster volume, reported at 5-minute intervals. **(Only available on Aurora MySQL and Postgres)**               |

</TabItem>
<TabItem value="Rds-Volume-Usage" label="Rds-Volume-Usage">

| Métrique        | Description                                                                                                                                                    |
| :-------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | VolumeBytesUsed | The amount of storage used by your Aurora DB instance, in bytes and then affecting the cost of your instance **(Only available on Aurora MySQL and Postgres)** |

Reviewer: ```suggestion
| VolumeBytesUsed | The amount of storage used by your Aurora DB instance, in bytes and then affecting the cost of your instance. **(Only available on Aurora MySQL and Postgres)** |
```

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |
| AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```

4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |

> | AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

Reviewer: ```suggestion
| AWSINSTANCENAME | Set the instance name (required) (can be defined multiple times)                                                            |                   | X           |
```

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```

4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |

> | AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

Reviewer: ```suggestion
| AWSINSTANCENAME | Set the instance name (required) (can be defined multiple times)                                                            |                   | X           |
```

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```

4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |

> | AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

Reviewer: ```suggestion
| AWSINSTANCENAME | Set the instance name (required) (can be defined multiple times)                                                            |                   | X           |
```

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```

4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |

> | AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

Reviewer: ```suggestion
| AWSINSTANCENAME | Set the instance name (required) (can be defined multiple times)                                                            |                   | X           |
```

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```

4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.

| Macro           | Description                                                                                                                | Valeur par défaut | Obligatoire |
|:----------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| AWSACCESSKEY    | Set AWS access key                                                                                                         |                   |             |
| AWSASSUMEROLE   | Set arn of the role to be assumed                                                                                          |                   |             |
| AWSCUSTOMMODE   | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |

> | AWSINSTANCENAME | Set the instance name (Required) (can be defined multipletimes)                                                            |                   | X           |

Reviewer: ```suggestion
| AWSINSTANCENAME | Set the instance name (required) (can be defined multiple times)                                                            |                   | X           |
```

> | AWSINSTANCETYPE | Set the instance type (Required) (Can be: 'cluster','instance')                                                            | instance          | X           |

Reviewer: ```suggestion
| AWSINSTANCETYPE | Set the instance type (required) (can be: 'cluster','instance')                                                            | instance          | X           |
```

> | AWSREGION       | Set the region name (Required)                                                                                             |                   | X           |

Reviewer: ```suggestion
| AWSREGION       | Set the region name (required)                                                                                             |                   | X           |
```
